bot/cogs/grades_data/grades_cog.py

```diff
@@ -33,19 +33,48 @@ class gradesCog(commands.Cog):
     def __init__(self, bot):
         self.bot = bot
         self.fileLines = []
+
         self.master_list = {}
         self.master_prof_list = {}
 
-    def process_Search(self, orig_query: str) -> str: #Primary search function and processing for a query. A query is generally defined by 
+        self.load_files()
+
+    def load_files(self):
+        temp = self.initialize('2014') # Store 2014
```

Comment:
this whole thing can be a loop that takes an iterable of strings, probably an iterable that is passed into the function